### ISSUE TITLE
ReferenceError: XMLHttpRequest is not defined

### ISSUE THREAD

**Issue by jeffbcross** [angular/zone.js#257]
As of 9041a3a5d8acf010b5a07dda577b8dd9920d4693, I'm getting an error when trying to load zones inside Universal: 

```
Projects/zone.js/dist/zone-microtask.js:2030
            Object.defineProperty(XMLHttpRequest.prototype, 'onreadystatechange', {
                                  ^

ReferenceError: XMLHttpRequest is not defined
    at canPatchViaPropertyDescriptor (Projects/zone.js/dist/zone-microtask.js:2030:28)
    at Object.apply (Projects/zone.js/dist/zone-microtask.js:1996:10)
    at Object.apply (Projects/zone.js/dist/zone-microtask.js:1593:30)
    at Object.<anonymous> (Projects/zone.js/dist/zone-microtask.js:61:19)
    at Object.<anonymous> (Projects/zone.js/dist/zone-microtask.js:64:31)
    at __webpack_require__ (Projects/zone.js/dist/zone-microtask.js:20:30)
    at Projects/zone.js/dist/zone-microtask.js:40:18
    at Object.<anonymous> (Projects/zone.js/dist/zone-microtask.js:43:10)
    at Module._compile (module.js:413:34)
    at Object.Module._extensions..js (module.js:422:10)
```

Offending line: https://github.com/angular/zone.js/blob/master/lib/patch/property-descriptor.ts#L42

CC @jteplitz602 @vicb

**Comment by jeffbcross** [comment]
I reverted the offending commits for now.

**Comment by jteplitz602** [comment]
Closed in 45a6bc1eaf23069d49c1f3e56d0b65b8b4118545
